tests/benchdnn/inputs/conv/test_conv_gpu_ci

```diff
@@ -1,83 +1,68 @@
 --reset
 
---mb=2,16
 --stag=any,axb
---dtag=any,axb
---alg=direct
 
 # Training
 ## Forward
 --dir=FWD_B
 --dt=f32,bf16,f16
---attr-post-ops=,relu,sum:0.25+relu:0.5
---batch=shapes_basic_gpu
+--attr-post-ops=,sum:0.25+relu:0.5
+--batch=shapes_ci_gpu
+--attr-post-ops=
+
 ## Backward
 --dir=BWD_D,BWD_WB
 --dt=f32,bf16
```

Comment:
We support f16 in training now, maybe that should be covered?

Reply:
Thanks, added.